modules/notification-workflowoperation/src/main/java/org/opencastproject/workflow/handler/notification/EmailWorkflowOperationHandler.java

```diff
@@ -104,6 +106,7 @@ public WorkflowOperationResult start(WorkflowInstance workflowInstance, JobConte
     String subject = applyTemplateIfNecessary(workflowInstance, operation, SUBJECT_PROPERTY);
     String bodyText = null;
     String body = operation.getConfiguration(BODY_PROPERTY);
+    Boolean isHTML = "true".equals(operation.getConfiguration(IS_HTML));
```

Comment:
This should use Boolean's .parse, no?

Reply:
Even better (and more user-friendly): [BooleanUtils.toBoolean(String)](https://commons.apache.org/proper/commons-lang/javadocs/api-3.0/org/apache/commons/lang3/BooleanUtils.html#toBoolean(java.lang.String))

Reply:
Yes, That is way better than what I wrote initially. Thank you.